src/components/Contact/Contact.jsx

```diff
@@ -0,0 +1,137 @@
+              >
+                <center>
+                  <FaGithubSquare style={{color:'#000000'}} />
+                  <br />
+                  Github
+                </center> 
+              </a>
+            
+            </IconContext.Provider>
+            <IconContext.Provider value={{ size: "4em" }}>
+              <a 
+                style={{padding:"5px"}} 
+                className="link"
+                href="https://www.facebook.com/Terasology"
+              >
+                <center> 
+                  <FaFacebookSquare style={{color:'#3b5998'}} />
+                  <br />
+                  Facebook
+                </center> 
+              </a>
+            </IconContext.Provider>
+            <IconContext.Provider value={{ size: "4em" }}>
+              <a 
+                style={{padding:"5px"}} 
+                className="link"
+                href="https://twitter.com/terasology"
+              >
+                <center> 
+                  <FaTwitterSquare style={{color:'#00acee'}} />
+                  <br />
+                  Twitter
+                </center> 
+              </a>
+            </IconContext.Provider>
+            <IconContext.Provider value={{ size: "4em" }}>
+              <a 
+                style={{padding:"5px"}}
+                className="link"
+                href="https://discord.gg/terasology"
+              >
+                <center> 
+                  <FaDiscord style={{color:'#7289da'}} />
+                  <br />
+                  Discord
+                  {' '}
+                </center> 
+              </a>
+            </IconContext.Provider>
+            <IconContext.Provider value={{ size: "4em" }}>
+              <a
+                style={{padding:"5px"}}
+                className="link"
+                href="https://www.youtube.com/user/blockmaniaTV"
+              >
+                <center> 
+                  <FaYoutubeSquare style={{color:'#c4302b'}} />
+                  <br />
+                  Youtube
+                  {' '}
+                </center> 
+              </a>
+            </IconContext.Provider>
+            <IconContext.Provider value={{ size: "4em" }}>
+              <a 
+                style={{padding:"5px"}}
+                className="link"
+                href="https://reddit.com/r/Terasology"
+              >
+                <center> 
+                  <FaRedditSquare style={{color:'#FF4301'}} />
+                  <br />
+                  Reddit
+                  {' '}
+                </center> 
+              </a>
+              <a 
+                style={{padding:"5px"}}
+                className="link"
+                href="http://webchat.freenode.net/?channels=terasology&uio=d4?channels=%23terasology&nick=Terasologist...&prompt=1&useUserListIcons=true"
+              >
+                <center>  
+                  <FaComments style={{color:'#7bb32e'}} />
+                  <br />
+                  Webchat
```

Comment:
It's IRC channel on freenode to be specific using Webchat's web UI. IRC would b a better name here.
Also, I don't know about the colourful icons, you should ask on Discord for community's input.